### PR TITLE
Spark-HPCC 7.0.0-snapshot

### PR DESCRIPTION
**Pull request by rpastrana**
- Corresponds to HPCC-Platform 7.0.0Beta

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
T E S T S
-------------------------------------------------------

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ spark-hpcc ---
[INFO] Building jar: /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-SNAPSHOT.jar
[INFO] 
[INFO] --- maven-assembly-plugin:2.4.1:single (spark-hpcc) @ spark-hpcc ---
[INFO] Building jar: /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-SNAPSHOT-jar-with-dependencies.jar
[INFO] 
[INFO] --- maven-install-plugin:2.4:install (default-install) @ spark-hpcc ---
[INFO] Installing /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-SNAPSHOT.jar to /root/.m2/repository/org/hpccsystems/spark-hpcc/7.0.0-SNAPSHOT/spark-hpcc-7.0.0-SNAPSHOT.jar
[INFO] Installing /home/hadoop/GIT/Spark-HPCC/DataAccess/pom.xml to /root/.m2/repository/org/hpccsystems/spark-hpcc/7.0.0-SNAPSHOT/spark-hpcc-7.0.0-SNAPSHOT.pom
[INFO] Installing /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-SNAPSHOT-jar-with-dependencies.jar to /root/.m2/repository/org/hpccsystems/spark-hpcc/7.0.0-SNAPSHOT/spark-hpcc-7.0.0-SNAPSHOT-jar-with-dependencies.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:35 min
[INFO] Finished at: 2018-06-07T13:15:42-04:00
[INFO] Final Memory: 91M/936M
[INFO] ------------------------------------------------------------------------

**Comment by rpastrana** [comment]
@richardkchapman please merge

File: DataAccess/pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.hpccsystems</groupId>
     <artifactId>spark-hpcc</artifactId>
-    <version>1.0.1-SNAPSHOT</version>
+    <version>7.0.0-SNAPSHOT</version>
```
Comment (richardkchapman) [inline DataAccess/pom.xml]: Why SNAPSHOT and not beta1?

**Comment by rpastrana** [comment]
T E S T S
-------------------------------------------------------

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0

[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ spark-hpcc ---
[INFO] Building jar: /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-beta-1.jar
[INFO] 
[INFO] --- maven-assembly-plugin:2.4.1:single (spark-hpcc) @ spark-hpcc ---
[INFO] Building jar: /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-beta-1-jar-with-dependencies.jar
[INFO] 
[INFO] --- maven-install-plugin:2.4:install (default-install) @ spark-hpcc ---
[INFO] Installing /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-beta-1.jar to /root/.m2/repository/org/hpccsystems/spark-hpcc/7.0.0-beta-1/spark-hpcc-7.0.0-beta-1.jar
[INFO] Installing /home/hadoop/GIT/Spark-HPCC/DataAccess/pom.xml to /root/.m2/repository/org/hpccsystems/spark-hpcc/7.0.0-beta-1/spark-hpcc-7.0.0-beta-1.pom
[INFO] Installing /home/hadoop/GIT/Spark-HPCC/DataAccess/target/spark-hpcc-7.0.0-beta-1-jar-with-dependencies.jar to /root/.m2/repository/org/hpccsystems/spark-hpcc/7.0.0-beta-1/spark-hpcc-7.0.0-beta-1-jar-with-dependencies.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:32 min
[INFO] Finished at: 2018-06-07T13:32:36-04:00
[INFO] Final Memory: 64M/1000M

**Comment by rpastrana** [comment]
@richardkchapman updated please merge

**Pull request merged**